models/Event.php

```diff
@@ -356,7 +356,7 @@ public function getRegistrationFieldNames($includeBase = true)
     public function showRegistrationForm()
```

Comment:
I don't know what you did, but the registration form just doesn't show at all.

Reply:
Are you perhaps trying to register for a past event?

Reply:
I see it is fixed now.